pkg/hash/v1alpha1/hash.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2019 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package hash
+
+import (
+	"crypto/md5"
+	"encoding/hex"
+	"encoding/json"
+	"github.com/pkg/errors"
+)
+
+// CalculateHash constructs the hash for any type of object
+func CalculateHash(obj interface{}) (string, error) {
```

Comment:
Can we rename this function as `Hash`

Reply:
@AmitKumarDas `Hash` is more appropriate than `New`